pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -176,6 +176,16 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		}
 	}
 
+	logger.Info("create cephConnection and defaultClientProfile for external mode")
```

Comment:
We need to only do this in case the csi operator is enabled, correct?

Reply:
no we want this for external mode as well

Reply:
Exactly, we can only create this upstream when csi operator is enabled.